### ISSUE TITLE
Does not work on my computer.

### ISSUE THREAD

**Issue by MarekKnapek** [tsoding/eepers#6]
Downloaded precompiled binary does not work on my Windows Server 2022 x64 computer. I'm connected to it via RDP. Other games work fine. The game does nothing (no window ever appears), it exits with zero exit code.

**Comment by rexim** [comment]
If you figure out what's the problem feel free to submit a PR :+1:

**Comment by iptoux** [comment]
> Downloaded precompiled binary does not work on my Windows Server 2022 x64 computer. I'm connected to it via RDP. Other games work fine. The game does nothing (no window ever appears), it exits with zero exit code.

Hey, you can compile itself on your windows machine. Its running fine on my side (self compiled) on my Windows 11.
Just follow this [Ada on windows and linux installation guide](https://www.noureddine.org/articles/ada-on-windows-and-linux-an-installation-guide#windows), but take the last releases from the repos [GNAT-FSF-builds](https://github.com/alire-project/GNAT-FSF-builds/releases) to install gnatmake/ada support. After this you need to Update the environ variables (can be done in windows settings) to the gnatmake, reboot.

Now open windows command line and go to the eepers repo and enter the following commands:

```powershell
windres eepers.rc -O coff -o eepers.res
gnatmake -k -mwindows -O3 -f -Wall -Wextra -gnat2022 eepers.adb -bargs -static -largs eepers.res -L./raylib/raylib-5.0_win64_mingw-w64/lib/ -l:libraylib.a -lwinmm -lgdi32 -static
```

**Comment by MarekKnapek** [comment]
Latest and greatest version 1.1 still does not work on  my computer.

**Comment by rexim** [comment]
@MarekKnapek what about 1.2?

**Comment by MarekKnapek** [comment]
No, both version 1.2 and version 1.3 do not work. In attachment there is API Monitor (http://www.rohitab.com/apimonitor) capture of version 1.3. [capture.zip](https://github.com/tsoding/eepers/files/14844553/capture.zip)
